src/erlcloud_aws.erl

```diff
@@ -967,3 +989,149 @@ error_msg( Message ) ->
 error_msg( Format, Values ) ->
     Error = iolist_to_binary( io_lib:format( Format, Values ) ),
     throw( {error, Error} ).
+
+
+
+%%====================================================================
+%% Test Functions
+%%====================================================================
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+
+
+default_profile_test_() ->
+    {setup, fun profiles_test_setup/0, fun profiles_test_cleanup/1,
+     ?_test(
+        ?assertMatch(
+           {ok, #aws_config{
+            access_key_id = "XXXXXXXXXXXXXXXXXXX2",
```

Comment:
Could you please move all this credential constants to variables/macros which are defined in one place?